src/user-sessions/user-sessions.c

```diff
@@ -31,10 +31,8 @@ static int run(int argc, char *argv[]) {
 
         if (streq(argv[1], "start")) {
                 r = unlink_or_warn("/run/nologin");
-                k = unlink_or_warn("/etc/nologin");
                 if (r < 0)
                         return r;
-                return k;
```

Comment:
We should return here, else it runs along and returns -EINVAL below.

```c
if (streq(...))
    return unlink_or_warn(...);
```

Reply:
Yikes.